draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -3613,15 +3613,15 @@ bytes=500-700,601-999
    (<xref target="attack.protocol.element.length"/>).
 </t>
 <t>
-   If a message is received that has multiple Content-Length header fields
-   with field values consisting of the same decimal value, or a single
-   Content-Length header field with a field value containing a list of
-   identical decimal values (e.g., "Content-Length: 42, 42"), indicating that
-   duplicate Content-Length header fields have been generated or combined by an
-   upstream message processor, then the recipient &MUST; either reject the
-   message as invalid or replace the duplicated field values with a single
-   valid Content-Length field containing that decimal value prior to
-   determining the message body length or forwarding the message.
+   If a message is received that has a Content-Length header field value
+   consisting of the same decimal value as a comma-separated list (<xref
+   target="abnf.extension"/>) -- for example, "Content-Length: 42, 42" --
+   indicating that duplicate Content-Length header fields have been generated
+   or combined by an upstream message processor, then the recipient &MUST;
+   either reject the message as invalid or replace the duplicated field
+   values with a single valid Content-Length field containing that decimal
+   value prior to determining the message body length or forwarding the
+   message.
```

Comment:
This already existed, but is replacing really the right language? A networking library should probably not alter the response before passing it on to some API endpoint. At least not for this reason.

Reply:
"forwarding the message" here means to another HTTP hop, not surfacing it to internal code. We don't specify that.